hbase-client/src/main/java/org/apache/hadoop/hbase/UserMetrics.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright The Apache Software Foundation
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hbase;
+
+import java.util.Map;
+import java.util.concurrent.atomic.LongAdder;
+
+import org.apache.hadoop.hbase.regionserver.MetricsUserSource;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+  * Encapsulates per-user load metrics.
+  */
+@InterfaceAudience.Public
+public interface UserMetrics {
```

Comment:
Evolving too? I think it should be...

Reply:
hbtop also provides a filtered read requests count metric from server load. That is missing here on the per user basis. Oversight?

Reply:
actually, we were not recording filteredReadRequest per user as a part of HBASE-15519 so skipped it. Let me add it  for this use-case

Reply:
Added in the latest commit now.